WordpressSync/debug.js

```diff
@@ -1,9 +1,10 @@
 //Loading environment variables
-const { Values } = require('../local.settings.json');
+const { Values } = require('../../local-wordpress-to-github.settings.json');
```

Comment:
the path for `local.settings.json` is the default for the Azure functions SDK.  This process is a way to have side-by-side compatibility with the file that Azure functions uses already.